manifest.webapp.in

```diff
@@ -13,6 +13,11 @@
   "default_locale": "en",
   "orientation": ["portrait-primary"],
   "permissions": {
+    // #if JSR_179 == 1
```

Comment:
This violates the constraint that unpreprocessed files be syntactically correct. Unfortunately, there's no way around it, since JSON doesn't allow comments. So this is the best we can do.